### ISSUE TITLE
How about calculating TEDS？

### ISSUE THREAD

**Issue by PC-god** [sachinraja13/TabStructNet#13]
Thanks your project.
According to lasted commit, I can calculate xml outputs. When I want to calculate TEDS metric, I don't find the corresponding function.
If you write corresponding function, please  indicate its location.
In the end, thanks very much.

**Comment by wangwen-whu** [comment]
Hi, I can  calculate xml outputs, too. And I have TEDS code, do you need? I can share with you. But I want to ask you the results you get on UNLV test data are like this?  
![image](https://user-images.githubusercontent.com/71618211/110889404-05d9f500-8329-11eb-8bab-8c6824421c18.png)
![image](https://user-images.githubusercontent.com/71618211/110889514-3b7ede00-8329-11eb-926e-0cfedb6bf743.png)

this jpg is in the /TabStructNet-master/trained_model/tab/result_jpg/AR_1024.jpg and  AR_1025.jpg, do you get the same results with me? I am wonder I get something wrong

**Comment by sachinraja13** [comment]
Can you please try to increase the DETECTION_NMS_THRESHOLD value to 0.5 or higher and test again?

**Comment by wangwen-whu** [comment]
> Can you please try to increase the DETECTION_NMS_THRESHOLD value to 0.5 or higher and test again?

Thank you very much! I change the  DETECTION_NMS_THRESHOLD value to 0.5 then I get all the bboxs ! This is really help me a lot!  May I ask you another question？ I try to use two GPUS to finetune the model on my own dataset,  change the GPU_COUNT=2 in tabnet.py, however i got this error seems like batch_size couldn't match the code. But if i change Image_PER_GPU=2 (or4) that is ok, could you please to help me use more than one GPU?
![image](https://user-images.githubusercontent.com/71618211/110942576-f1bbe500-8374-11eb-90a8-044715411a78.png)

**Comment by sachinraja13** [comment]
Hi, I will try and figure this out and will update you.

**Comment by wangwen-whu** [comment]
Thank you very much! Looking forward to your reply~~

**Comment by yihaoding** [comment]
@PC-god @wangwen-whu Hi, is there any comments talking about how to evalutate the output xml files? Thanks for your help.

**Comment by anishdulal** [comment]
Hi @wangwen-whu, can you please share the code to generate the TEDS metric?

**Comment by wangwen-whu** [comment]
Hi，the TEDS code is mentioned here https://github.com/ibm-aur-nlp/PubTabNet/tree/master/src，need html style to be the input, next month, I will upload a transform code  in my work:  WTW dataset .  You can use it  if you need it too.

**Comment by wangwen-whu** [comment]
Hi，the TEDS code is mentioned here https://github.com/ibm-aur-nlp/PubTabNet/tree/master/src
  it need html style to be the input, next month, I will upload a transform code in my work: WTW dataset . You can use it if you need it too.

**Comment by anishdulal** [comment]
@wangwen-whu, thank you very much!

**Comment by anishdulal** [comment]
Hi @wangwen-whu , were you able to solve the issue with training on two GPUs? My GPU is just 8GB and it won't fit two images so I can't make IMAGE_PER_GPU=2 but I have access to multiple GPUs of 8GB.  I am now stucked at the problem you mentioned earlier.
